Cargo.toml

```diff
@@ -17,14 +17,17 @@ futures-util = "0.3"
 giganto-client = { path ="giganto-client" }
 humantime = "2.1"
 humantime-serde = "1"
+libc = "0.2.140"
```

Comment:
Any reason to use the specific version?

Reply:
Yes. This is the most recent version and I just followed the line here : https://crates.io/crates/libc

Reply:
In order to follow any updates, we just have to specify 0.2 without 140